src/mbgl/tile/tile_data.hpp

```diff
@@ -82,6 +84,16 @@ class TileData : private util::noncopyable {
     virtual void redoPlacement(PlacementConfig, const std::function<void()>&) {}
     virtual void redoPlacement(const std::function<void()>&) {}
 
+    virtual void queryRenderedFeatures(
+            std::unordered_map<std::string, std::vector<std::string>>&,
```

Comment:
They weren't named to avoid unused variable warnings.
Is this better? ea80c70bca770fe9901403a1d32694cfb3e958b3
or should I move that to raster_tile_data.cpp?